templates/zerver/help/display-emojis-as-text.md

```diff
@@ -0,0 +1,13 @@
+## Display emojis as text
+
+You might want to display emojis as text to increase readability. For example,
+when a heart emoji is displayed as text, it looks like `:heart:`.
+
+{settings_tab|display-settings}
+
+2. Under **Emoji Style**, select the option labeled **Plain text**.
+
+!!! tip ""
```

Comment:
I would just put this under a related articles heading, with "You may also be interested in ...".